spec.bs

```diff
@@ -1189,41 +1192,155 @@ partial interface Navigator {
+          1. Jump to the step labeled <i><a href=#abort-update>Abort update</a></i>
+            if any of the following conditions hold:
+            * |parsedURL| is failure;
+            * |parsedURL| is not [=same origin=] with |ig|'s [=interest group/owner=];
+            * |parsedURL| [=includes credentials=];
+            * |parsedURL| [=url/fragment=] is not null.
+          1. Set |ig|'s |interestGroupField| to |parsedURL|.
+          
+        <dt>"`trustedBiddingSignalsKeys`"
+        <dd>
+        1. If |value| is a [=list=] of [=strings=],
+          set |ig|'s [=interest group/trusted bidding signals keys=] to |value|.
+        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+
+        <dt>"`ads`"
+        <dt>"`adComponents`"
+        <dd>
+        1. For each |groupMember| and |interestGroupField| in the following table
+          <table class="data">
+            <thead><tr><th>Group member</th><th>Interest group field</th></tr></thead>
+            <tr>
+              <td>"`ads`"</td>
+              <td>[=interest group/ads=]</td>
+            </tr>
+            <tr>
+              <td>"`adComponents`"</td>
+              <td>[=interest group/ad components=]</td>
+            </tr>
+          </table>
+          1. If |key| is not |groupMember|, [=iteration/continue=].
+          1. If |value| is not a [=list=] of {{AuctionAd}}, jump to the step
+            labeled <i><a href=#abort-update>Abort update</a></i>.
+          1. [=list/For each=] |ad| of |value|:
+            1. Let |igAd| be a new [=interest group ad=].
+            1. Let |renderURL| be the result of running the [=URL parser=] on
+              |ad|["{{AuctionAd/renderURL}}"].
+            1. Jump to the step labeled <i><a href=#abort-update>Abort update</a></i>
+              if any of the following conditions hold:
+              * |renderURL| is failure;
+              * |renderURL| [=url/scheme=] is not "`https`";
+              * |renderURL| [=includes credentials=].
+            1. Set |igAd|'s [=interest group ad/render url=] to |renderURL|.
+            1. If |ad|["{{AuctionAd/metadata}}"] [=map/exists=], then let
+              |igAd|'s [=interest group ad/metadata=] be the result of
+              [=serializing a JavaScript value to a JSON string=], given |ad|["{{AuctionAd/metadata}}"].
+              If this can [=exception/throws=],  jump to the step
+              labeled <i><a href=#abort-update>Abort update</a></i>.
+            1. [=list/Append=] |igAd| to |ig|'s |interestGroupField|.
+              
+        </dl>
+              
+    1. Set |ig|'s [=interest group/next update after=] to the current time plus 24 hours.
+    1. [=list/Replace=] |originalInterestGroup| with |ig| in the browser's
+      [=interest group set=].
+    1. <i id=abort-update>Abort update</i>: We jump here if some part of the
+      [=interest group=] update failed.  Continue to the next [=interest group=] update.

```

Comment:
This "continue" corresponds to the inner loop right? In that case, should we link to `[=iteration/Continue=]`?